{{cookiecutter.project_slug}}/server/{{cookiecutter.project_slug}}/settings.py

```diff
@@ -242,7 +242,7 @@
 # Django will look for client-side build files in this directory
 STATICFILES_DIRS = [
     os.path.join(BASE_DIR, "..", "client", "build"),
-    os.path.join(BASE_DIR, "..", "client", "build", "static"),
+    os.path.join(BASE_DIR, "..", "client", "build", "media-files"),
```

Comment:
The media-files directory should _not_ be included in `STATICFILES_DIRS`. This change will also break the front end app!

"Static" files are the mostly-unchanging files served by the web application. Things like: CSS, JS, fonts, and hard-coded images.

"Media" files are dynamic files, typically uploaded by users.

They serve different purposes. They can both be stored on S3, but their configuration needs to be kept separate!

Reply:
Sorry I had asked for this change I may have not made it clear @jdephil I thought it was line 229 that was set incorrectly!